src/app.vue

```diff
@@ -109,6 +109,10 @@ if (import.meta.env.MODE !== 'production') {
   })
 }
 
+function logError(error) {
+  console.error(error)
+}
+
```

Comment:
```suggestion
```
Is this used somewhere?

Reply:
yep, there https://github.com/aeternity/aescan/blob/89be1733935d8feb7dcaca41cbdd7a27f5bfd90c/src/app.vue#L65